src/sagas/authSaga.js

```diff
@@ -6,16 +6,22 @@ import {
   loginSuccess,
   loginError,
   getErrors,
+  setUser
 } from "../actions/authActions";
 
 const { open } = notification;
 
 function* loginSaga(action) {
   try {
     const response = yield call(authService.login, action.payload);
-    const { token, userId } = response;
+    const { token, userId, name, surname } = response;
     localStorage.setItem("accessToken", token);
     yield put(loginSuccess(userId));
+    yield put(setUser( {
```

Comment:
this will work after login But did you try to refresh page I don't see any call that will get this data back

Reply:
will test once again, but 99% sure that persistStore saves data between refreshes